game_view.cpp

```diff
@@ -279,10 +279,15 @@ void game_view::draw_player_coords() noexcept
     std::string str_player_coords;
     for(int i = 0; i != static_cast<int>(v_player.size()); i++) {
         player p = v_player[static_cast<unsigned int>(i)];
-        str_player_coords += "Player " + p.get_ID() + " x = " + std::to_string(trunc(p.get_x()));
-        str_player_coords += "\nPlayer " + p.get_ID() + " y = " + std::to_string(trunc(p.get_y()));
+        str_player_coords += "Player " + p.get_ID() + " x = " + std::to_string(trunc(get_x(p)));
+        str_player_coords += "\nPlayer " + p.get_ID() + " y = " + std::to_string(trunc(get_y(p)));
         str_player_coords += "\n\n";
     }
+    food f = m_game.get_food()[0];
+    str_player_coords += "Food x = " + std::to_string(trunc(get_x(f)));
+    str_player_coords += "y = " + std::to_string(trunc(get_y(f)));
+    str_player_coords += "\n\n";
```

Comment:
Do we want to keep this? I believe it may still be useful for dev for now but it is placed awkwardly.

Reply:
yeah maybe in another issue